extensions/extensions-configuration-store-vault/pom.xml

```diff
@@ -87,5 +87,10 @@
             <artifactId>foundation-configuration-store</artifactId>
             <version>${version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive</artifactId>
+            <version>2.8.3.Final</version>
```

Comment:
S: Let's use the parent POM version. I think you can just omit it as it's defined in a `dependencyManagement`, or set it to `${version.quarkus}`